rpcs3/rpcs3qt/gs_frame.cpp

```diff
@@ -187,20 +187,6 @@ void gs_frame::keyPressEvent(QKeyEvent *keyEvent)
 			return;
 		}
 		break;
-	case Qt::Key_S:
-		if (keyEvent->modifiers() == Qt::ControlModifier && !m_disable_kb_hotkeys && !Emu.IsStopped())
-		{
-			Emu.Stop();
-			return;
-		}
-		break;
-	case Qt::Key_R:
```

Comment:
ctrl+R can mean now what ctrl+E currently does which is Resume emulation.

Reply:
ill do it tomorrow, would turn into a mess if i kept relying on github's web editor

Reply:
done